w04-associations-and-auth/d3-homework/README.md

```diff
@@ -0,0 +1,122 @@
+# Project 1 Planning
+
+## [Requirements](https://github.com/sf-wdi-22-23/modules/tree/master/w04-associations-and-auth/project-1)
```

Comment:
@tgaff Can you double check this after she merges her PR and then merge this if it looks okay?